lib/lineage.js

```diff
@@ -229,10 +257,11 @@ module.exports = {
    * @param docs The array of docs to hydrate
    * @returns Promise
    */
+   //TODO @gareth should I add patients to this?
```

Comment:
Yeah probably best to be consistent about what "hydrated" means...

Reply:
I've raised #4003 so I can do this later in 2.15 and not be held back doing more important things